Tests/CommandsTests/APIDiffTests.swift

```diff
@@ -426,65 +424,74 @@ class APIDiffTestCase: CommandsBuildProviderTestCase {
             let revision = try repo.resolveRevision(identifier: "1.2.3")
 
             let baselineDir = fixturePath.appending("Baselines")
-            let fooBaselinePath = baselineDir.appending(components: revision.identifier, "Foo.json")
+            let fooBaselinePath: AbsolutePath
+            if buildSystem == .swiftbuild {
+                fooBaselinePath = baselineDir.appending(components: revision.identifier, "Foo")
+            } else {
+                fooBaselinePath = baselineDir.appending(components: revision.identifier, "Foo.json")
+            }
 
-            try localFileSystem.createDirectory(fooBaselinePath.parentDirectory, recursive: true)
-            try localFileSystem.writeFileContents(
-                fooBaselinePath,
-                string: "Old Baseline"
-            )
+            var initialTimestamp: Date?
+            await expectThrowsCommandExecutionError(
+                try await execute(["diagnose-api-breaking-changes", "1.2.3",
+                             "--baseline-dir", baselineDir.pathString],
+                            packagePath: packageRoot,
+                                 buildSystem: buildSystem)
+            ) { error in
+                expectMatch(error.stdout, .contains("1 breaking change detected in Foo"))
+                expectMatch(error.stdout, .contains("💔 API breakage: func foo() has been removed"))
+                #expect(localFileSystem.exists(fooBaselinePath))
```

Comment:
suggestion: instead of using `localFileSystem`, which is defined in Swift Tools Support Core (STSC), can we use `FileManager` instead to remove our dependency on STSC?

This comment applies to all other code changes that use `localFileSystem`